mne/preprocessing/tests/test_maxwell.py

```diff
@@ -38,10 +38,8 @@
 sss_ctc_fname = pre + 'crossTalk_raw_sss.fif'
 sss_trans_default_fname = pre + 'transDefault_raw_sss.fif'
 sss_trans_sample_fname = pre + 'transSample_raw_sss.fif'
-sss_st1FineCalCrossTalkRegIn_fname = \
-    pre + 'st1FineCalCrossTalkRegIn_raw_sss.fif'
-sss_st1FineCalCrossTalkRegInTransSample_fname = \
-    pre + 'st1FineCalCrossTalkRegInTransSample_raw_sss.fif'
+sss_st1FineCalCrossTalkRegIn_fname = pre + 'st1FineCalCrossTalkRegIn_raw_sss.fif'  # noqa: E501,N816
+sss_st1FineCalCrossTalkRegInTransSample_fname = pre + 'st1FineCalCrossTalkRegInTransSample_raw_sss.fif'  # noqa: E501,N816
```

Comment:
these are ugly and *should* get changed IMO, but since they're in a test I'm not inclined to fight about it.

Reply:
Agreed they are pretty bad (and inconsistent with the rest of our code base -- thanks `flake8`!), I'll change them here

Reply:
Okay @drammock did some name unification

Reply:
`mne/preprocessing/tests/test_maxwell.py:59:30: E128 continuation line under-indented for visual indent`